versions/2.0/schema/digitized_audioopticaldisc.json

```diff
@@ -36,16 +36,27 @@
           "$ref": "fields#technicalFilename"
         },
         "extension": {
-          "$ref": "fields#technicalExtensionForAudio"
+          "enum": [
+            "bin",
+            "BIN",
+            "WAV",
+            "wav"
+          ]
         },
         "dateCreated": {
           "$ref": "fields#technicalDateCreated"
         },
         "fileFormat": {
-          "$ref": "fields#technicalFileFormatForAudio"
```

Comment:
I can't find anything in https://github.com/MediaArea/MediaInfoLib/tree/a83901645b5327ac175976894d730d1590baf552/Source/Resource/Text/DataBase